systems/plants/RigidBodyManipulator.h

```diff
@@ -212,6 +212,8 @@ class DLLEXPORT_RBM RigidBodyManipulator
   DrakeCollision::ElementId addCollisionElement(const RigidBody::CollisionElement& element, const std::shared_ptr<RigidBody>& body, std::string group_name);
 
   void updateCollisionElements(const std::shared_ptr<RigidBody>& body);
+  void getTerrainPoints(int body_id, Eigen::MatrixXd &terrain_points);
+  void getTerrainPoints(const std::shared_ptr<RigidBody>& body, Eigen::MatrixXd &terrain_points);
```

Comment:
I think this method should take a RigidBody& as an argument. If we decide otherwise, we should check for null pointers in the implementation.